src/form-entry.js

```diff
@@ -0,0 +1,39 @@
+import store from './data/store.js';
+
+const form = document.getElementById('product-form');
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    const formData = new FormData(form);
+
+    const image = formData.get('image');
+    getBase64(image, (base64Url) => {
+
+        const product = {
+            code: formData.get('code'),
+            name: formData.get('name'),
+            image: base64Url,
+            category: formData.get('category'),
+            price: +formData.get('price'),
+            description: formData.get('description')
+        };
+
+        store.addProduct(product);
+
+        alert('product saved');
+
+        form.reset();
+
```

Comment:
This is super nit-picky, but you should get rid of white space like lines 27 and 29 - it's definitely good to have blank lines in between parts of code, but with just brackets like this you shouldn't have that much whitespace.